include/rmm/detail/dynamic_load_runtime.hpp

```diff
@@ -116,6 +121,28 @@ struct async_alloc {
     return runtime_supports_pool and driver_supports_pool;
   }
 
+  /**
+   * @brief Check whether the specified `cudaMemAllocationHandleType` is supported on the present
+   * CUDA driver/runtime version.
+   *
+   * @note This query was introduced in CUDA 11.3 so on CUDA 11.2 this will only return true for
```

Comment:
The referent here gets obscured by the "this query" at the beginning.
```suggestion
   * @note This query was introduced in CUDA 11.3 so on CUDA 11.2 this function will only return true for
```